finetune/util/table_labeler.py

```diff
@@ -389,6 +388,7 @@ def predict_from_file(
         scheduler: Scheduler,
         return_negative_confidence=False,
         config_overrides=None,
+        sobj_url=None,
     ):
         etl = get_etl_from_file(model_file_path)
```

Comment:
i'm a bit concerned here with the interaction of the cache hit or miss. in cusv2 non table predict calls we pass a None value to model_file_path when the model is in cache, looks like we can't do that here due to the etl load, so we may be reading from blob more often.

Reply:
Want to confirm with Ben, but this is just a thin wrapper for `Model.load(model_file_path, key='etl')`.  So I think we could considering wiring this through a new Scheduler.load method that would also hit a cache?

Reply:
I reviewed this on Friday but forgot to press go so this may have come through in a strange order. Agree with Madison that his suggestion is definitely an option. Just having a Non-model cache as part of the scheduler. Or keeping it as a separate cache is also fine.